.changeset/early-dots-drive.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-org': patch
```

Comment:
Not sure if patch is needed here since it doesn't change existing behaviour in a breaking way.